docs_user/modules/openstack-edpm_adoption.adoc

```diff
@@ -231,6 +275,7 @@ spec:
     - install-certs
     - libvirt
     - nova-compute-extraconfig
+    - nova-custom-ceph
```

Comment:
havign two dataplane service for nova in the same node set is not supported
its only functional because of a bug which weneed to fix before ga
so i know you were doing this for testing but i would avoid doing this even for testing as this pattern will stop working in the next week or two.

Reply:
thanks for explaining, I'm merging these

Reply:
done